pyproject.toml

```diff
@@ -1,21 +1,28 @@
 [tool.isort]
 profile = "black"
 sections = [
-  "FUTURE",
-  "STDLIB",
-  "THIRDPARTY",
-  "ARCHSPEC", "LLNL", "FIRSTPARTY",
-  "LOCALFOLDER",
-]
+    "FUTURE",
+    "STDLIB",
+    "TYPING",
+    "THIRDPARTY",
+    "COMPAT",
+    "ARCHSPEC", "RUAMEL", "LLNL", "FIRSTPARTY",
+    "LOCALFOLDER",
+  ]
 known_first_party = "spack"
 known_archspec = "archspec"
+known_ruamel = "ruamel"
+known_compat = "compat"
 known_llnl = "llnl"
+known_third_party = "six"
+known_typing = ["typing", "typing_extensions"]
```

Comment:
These libraries are all third-party libraries, and don't need an additional import section. Why are these changes necessary?

Reply:
Thanks so much!! I have removed these individual import sections and rolled them all into `"THIRDPARTY"`.